added smaller, linkable url

dattpatel123 opened the pull request:
fixed issue #16. Also not sure why it's showing changes to truncateStringAtWord(), I only touched createTable(). would be nice if you could tell me as im new to git. Thanks!

luizvbo commented:
Hi! Thanks for contributing. I hope the repo was useful.

You're seeing changes to truncateStringAtWord() because of line ending differences, not actual code changes.

When you opened the file, your text editor (or operating system) likely converted the file's line endings from what Git expected (typically LF, or Line Feed, common on Linux/macOS) to CRLF (Carriage Return Line Feed, common on Windows). 

You can fix that by running:
```
git config --global core.autocrlf true
```

Could you fix it in your PR?

dattpatel123 commented:
Hi I changed the setting but don't think the fix applied. originally I couldnt push cause no changes were made so I just added an extra space and pushed but looks like the previous issue is still there. what should I do?
